pennylane_lightning/core/src/observables/Observables.hpp

```diff
@@ -511,11 +511,13 @@ class SparseHamiltonianBase : public Observable<StateVectorT> {
             ss << "{" << d.real() << ", " << d.imag() << "},"
                << "\n";
         ss << ",\n'indices' : \n";
-        for (const auto &i : indices_)
-            ss << i << std::endl;
+        for (const auto &i : indices_) {
+            ss << i;
```

Comment:
do we need a `space` and a `comma` after each element?

Reply:
Yes. To keep formatting uniform, and ease copy paste (to NumPy for example), I would put each vector on one line.